src/v/cluster/partition_probe.cc

```diff
@@ -267,6 +267,142 @@ void replicated_partition_probe::setup_public_metrics(const model::ntp& ntp) {
               .aggregate({sm::shard_label, partition_label}),
           });
     }
+
+    setup_public_scrubber_metric(ntp);
+}
+
+void replicated_partition_probe::setup_public_scrubber_metric(
+  const model::ntp& ntp) {
+    namespace sm = ss::metrics;
+
+    // No point in setting up the scrubber metrics if there's no
+    // archival metadata STM to pull values from.
+    if (!_partition.archival_meta_stm()) {
+        return;
+    }
+
+    auto ns_label = metrics::make_namespaced_label("namespace");
+    auto topic_label = metrics::make_namespaced_label("topic");
+    auto partition_label = metrics::make_namespaced_label("partition");
+
+    const std::vector<sm::label_instance> common_labels{
+      ns_label(ntp.ns()),
+      topic_label(ntp.tp.topic()),
+      partition_label(ntp.tp.partition()),
+    };
+
+    auto anomaly_type_label = metrics::make_namespaced_label("type");
+    auto severity_type_label = metrics::make_namespaced_label("severity");
+
+    auto generate = [&, this](
+                      ss::sstring name,
+                      ss::sstring description,
+                      ss::sstring severity,
+                      auto extractor) -> ss::metrics::metric_definition {
+        auto labels = common_labels;
+        labels.push_back(anomaly_type_label(std::move(name)));
+        labels.push_back(severity_type_label(std::move(severity)));
+
+        return sm::make_gauge(
+                 "anomalies",
+                 [this, extractor = std::move(extractor)] {
+                     if (
+                       !_partition.is_elected_leader()
+                       || !_partition.archival_meta_stm()) {
+                         return size_t{0};
+                     }
+
+                     const auto& anomalies = _partition.archival_meta_stm()
+                                               ->manifest()
+                                               .detected_anomalies();
+
+                     return extractor(anomalies);
+                 },
+                 sm::description(std::move(description)),
+                 labels)
+          .aggregate({sm::shard_label, partition_label});
+    };
+
+    _public_metrics.add_group(
+      prometheus_sanitize::metrics_name("cloud_storage"),
+      {
+        generate(
+          "missing_partition_manifest",
+          "Count of missing partition manifest anomalies for the topic",
```

Comment:
@VladLazar what about  a global number of this type of anomaly for the whole cluster? Should't we have that too?

Reply:
We've discussed Redpanda doing cluster-wide aggregation in the team. It's not going to happen for 23.3 since it's quite a bit of work. We don't really have a great place to store this metadata at the cluster level.

The prometheus query will aggregate by cluster, although that has some downsides: the result of the query may flap due to leadership moves.